Packs/Troubleshoot/Scripts/TroubleshootExecutePlaybookByAlertQuery/TroubleshootExecutePlaybookByAlertQuery.py

```diff
@@ -217,12 +239,19 @@ def set_playbook_on_alerts(playbook_id: str, alert_ids: list, playbooks_dict: di
         return
 
     demisto.debug(f"Start setting playbook {playbook_id} on alerts {alert_ids}.")
-    command_results = demisto.executeCommand(
-        "core-api-post", {"uri": "/xsoar/inv-playbook/new", "body":
-                          {"playbookId": playbook_id, "alertIds": alert_ids, "version": -1}})
+    if flag_pending_idle:
+        command_results = demisto.executeCommand(
+            "core-api-post", {"uri": "/xsoar/inv-playbook/new", "body":
+                              {"playbookId": playbook_id, "alertIds": alert_ids, "version": -1}})
 
-    demisto.debug(f"Results of setting playbook {playbook_id} on alerts {alert_ids}:\n{command_results}")
-    handle_results(command_results, playbook_id, alert_ids, results_summary)
+        demisto.debug(f"Results of setting playbook {playbook_id} on alerts {alert_ids}:\n{command_results}")
+        handle_results(command_results, playbook_id, alert_ids, results_summary)
```

Comment:
Why are the 2 lines repeating this is the same functionality for the if and the else.